repo/packages/D/datastax-dse/200/resource.json

```diff
@@ -0,0 +1,62 @@
+{
+  "assets": {
+    "container": {
+      "docker": {
+        "dse": "dspn/10517a0c8e9b11e79ef5600308a463aa-3:5.1.2"
+      }
+    },
+    "uris": {
+      "jre-tar-gz": "https://downloads.mesosphere.com/java/server-jre-8u162-linux-x64.tar.gz",
+      "libmesos-bundle-tar-gz": "https://downloads.mesosphere.com/libmesos-bundle/libmesos-bundle-master-28f8827.tar.gz",
```

Comment:
libmesos-bundle seems to be the old? why not libmesos-bundle-1.11.0.tar.gz?

Reply:
Externals weren't updated yet. Which is fine. We should just make sure to update them in the next release.